tests/DbAuth.py

```diff
@@ -38,12 +38,11 @@ class DbAuthTestCase(unittest.TestCase):
     """A test case for DbAuth."""
 
     def setUp(self):
-        self.pol = Policy(os.path.join(ROOT, "testDbAuth.paf"))
```

Comment:
That was me misunderstanding things when I tried to fix a leak in 26518a12bda179b6b86a2afd7bd1b7e285be3fed